docs/DeveloperGuide.md

```diff
@@ -1336,8 +1330,14 @@ testers are expected to do more *exploratory* testing.
 
     1. Prerequisites: Filter contacts using the `find` command.
 
-    1. Test case: `mailingList` as above<br>
-        Expected: Same result as exporting full contact list, but with only the contacts in the filtered list
+    1. Test case: variations on `mailingList` as above<br>
+        Expected: Same result as exporting full contact list, but with only the contacts displayed in the filtered list
+
+1. Export Empty contact list
+        1. Prerequisites: Filter contacts using the `find` command.
```

Comment:
Might want to adjust the spacing here

Reply:
yeah if not there will be inconsistencies in numbering.